docs/sources/query/log_queries/_index.md

```diff
@@ -694,3 +711,24 @@ the result will be
 {level="info"} {"app": "other-service", "level": "info", "method": "GET", "path": "/", "host": "grafana.net", "status": "200"}
 ```
 
+## Pipeline Errors
+
+There are multiple reasons which cause pipeline processing errors, such as:
+
+- A numeric label filter may fail to turn a label value into a number
+- A metric conversion for a label may fail.
+- A log line is not a valid json document.
+- etc...
+
+When those failures happen, Loki won't filter out those log lines. Instead they are passed into the next stage of the pipeline with a new system label named `__error__`. The only way to filter out errors is by using a label filter expressions. The `__error__` label can't be renamed via the language.
```

Comment:
```suggestion
When label filter failures happen, Loki can't filter out log lines. Instead they are passed into the next stage of the pipeline with a new system label named `__error__`. The only way to filter out errors is by using a label filter expression. The `__error__` label can't be renamed via the language.
```

Reply:
Also, need some clarity on what you mean by "can't be renamed via the language."  What language are you talking about here?

Reply:
referring to LogQL, i updated the line to make it clear.